test/integration/mirror/serverspec/kafka_mirror_maker_spec.rb

```diff
@@ -0,0 +1,94 @@
+# coding: UTF-8
+
+require 'spec_helper'
+
+describe user('kafka') do
+  it { should exist }
+  it { should belong_to_group 'kafka' }
+end
+
+describe service('kafka-mirror-maker') do
+  it { should be_running   }
+end
+
+# Kafka mirror JMX
+# Not yet enabled in launcher
+#describe port(9999) do
+#  it { should be_listening }
+#end
+
+describe file('/opt/kafka/config/mirror_target.properties') do
+  it { should be_file }
+  it { should be_owned_by 'kafka' }
+  it { should be_grouped_into 'kafka' }
+  #it { should contain 'bogus.for.testing=server' }
+end
+
+describe file('/opt/kafka/config/mirror_source1.properties') do
+  it { should be_file }
+  it { should be_owned_by 'kafka' }
+  it { should be_grouped_into 'kafka' }
+  #it { should contain 'bogus.for.testing=server' }
+end
+
+describe file('/opt/kafka/config/mirror-log4j.properties') do
+  it { should be_file }
+  it { should be_owned_by 'kafka' }
+  it { should be_grouped_into 'kafka' }
+  #it { should contain 'bogus.for.testing=log4j' }
+end
+
+describe 'kafka mirror' do
+
+  it 'should own all files' do
+    # Ensure we reload ruby's usernames/groups
+    Etc.endgrent
+    Etc.endpwent
+    Dir["/opt/kafka/**/*"].each do |filePath|
+      expect(Etc.getpwuid(File.stat(filePath).uid).name).to eq("kafka")
+      expect(Etc.getgrgid(File.stat(filePath).gid).name).to eq("kafka")
+    end
+  end
+
+  it 'should be able to read/write from a topic through a mirror' do
```

Comment:
Ugh, I wrote this test originally but now that we've had it for awhile this seems pretty intense for chef. I would be fine without if as long as we have tests that show we configure and start things properly